src/index.html

```diff
@@ -9,6 +9,20 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header" id="home">
+      <a href="#home"><img src="./images/logo.png" alt="logothype"></a>
+      <nav class="nav_list">
+        <ul class="links">
+          <li><a href="index.html" class="link is-active">Apple</a></li>
+          <li><a href="index.html" class="link">Samsung</a></li>
+          <li><a href="index.html" class="link">Smartphones</a></li>
+          <li data-qa="hover"><a href="index.html" class="link colored">Laptops & Computers</a></li>
```

Comment:
```suggestion
          <li data-qa="hover"><a href="index.html" class="link">Laptops & Computers</a></li>
```
colored redundant, change color with only with is-active and hover